docs/content/docs/concepts/dataobject-aqueduct.md

```diff
@@ -100,7 +100,7 @@ SharedDirectory.
 
 {{% callout important "See also" %}}
 
-- [Creating and storing distributed data structures](./dds.md#creating-and-storing-distributed-data-structures)
+- [Creating and storing distributed data structures](../dds/#creating-and-storing-distributed-data-structures)
```

Comment:
```suggestion
- [Creating and storing distributed data structures](./dds/#creating-and-storing-distributed-data-structures)
```

Reply:
That doesn't work. Perhaps because it's not an .md link?